national-resources/norway.md

```diff
@@ -0,0 +1,336 @@
+
+## Domain-specific infrastructures or resources 
+<!--- e.g. human data, covid-19. Please, only add domain-specific resources that you think don't fit in the table at the bottom--->
+### National infrastructures
+* [The Norwegian Institute of Public Health’s Antibiotic Committee (Folkehelseinstituttets antibiotikakomitéen)](https://www.fhi.no/sv/antibiotikaresistens/antibiotikakomiteen/). The [FHI provides a general information and resources on antibiotic resistance](https://www.fhi.no/sv/antibiotikaresistens/), useful for both public, researchers, and health service professionals
+* [CAMRIA (Combatting Antimicrobial Resistance with Interdisciplinary Approaches)](https://camria.w.uib.no/), launched at the University of Bergen (UiB), is a research centre with multiple partners (Haukeland University Hospital and Stavanger University Hospital) that work together to address AMR topics in a context of microbial genomics and related areas.
+* [The National Advisory Unit on Detection of Antimicrobial Resistance (K-res)](https://unn.no/fag-og-forskning/k-res) provides laboratory-based services for the detection and surveillance of antimicrobial resistance in Norway. The unit offers diagnostic and advisory services, trains healthcare professionals, and advises on appropriate antibiotic distribution and use.
+* [Norwegian Society for Medical Microbiology (Norsk forening for medisinsk mikrobiologi (NFMM)](https://www.legeforeningen.no/foreningsledd/fagmed/norsk-forening-for-medisinsk-mikrobiologi/) is a national association to bring together experts from medical microbiology fields for knowledge-exchange. NFMM also provides a [database of clinical laboratories that take care of infectious disease, pathogen, and related sampling](https://www.legeforeningen.no/foreningsledd/fagmed/norsk-forening-for-medisinsk-mikrobiologi/fag/kliniske-laboratorier-i-norge/).
+
+### Institutional initiatives on antimicrobial resistance (AMR)
+There are national research projects focused towards pathogenomics to help improve agriculture, food, public health, environment, and biodiversity in Norway. Also, higher education institutions and industrial organisations have forums and research centres dedicated to addressing the topics of antimicrobial resistance and pandemic preparedness.
+* [PathoSeq](https://nofima.com/projects/pathoseq/), a project on sequencing pathogenic bacteria for food safety. More information is available on the [University of Oslo PathoSeq page](https://www.jus.uio.no/ifp/english/research/projects/nrccl/pathoseq/).
+* [Interdisciplinary Forum for Research on Antimicrobial resistance (INFRA)](https://www.med.uio.no/she/english/network/INFRA/index.html), University of Oslo
```

Comment:
For this and the one below, perhaps we need a sentence on what they are?

Reply:
Do you mean for PathoSeq and INFRA? We think titles and websites are sufficient here, as descriptions are mentioned on their homepages.